__tests__/compilers/compatability.test.tsx

```diff
@@ -495,4 +494,24 @@ ${JSON.stringify(
       "
     `);
   });
+
+  it('compiles inline html', () => {
+    const md = `Inline html: <small>_string_</small>`;
+
+    const rmdx = mdx(rdmd.mdast(md));
+    expect(rmdx).toMatchInlineSnapshot(`
+      "Inline html: <small>*string*</small>
+      "
+    `);
+  });
+
+  it('compiles tag-like syntax', () => {
+    const md = `Inline: <what even is this>`;
+
+    const rmdx = mdx(rdmd.mdast(md));
+    expect(rmdx).toMatchInlineSnapshot(`
+      "Inline: <what even is this>
```

Comment:
This syntax _would_ fail to render in MDX though, right?

Reply:
Yes, but it's much better than what it was doing:

```
Inline: <what even="" is="" this="" />
```

Greg pointed out this is confusing the AI prompt.

I had a [PR up for an possible fix](https://github.com/readmeio/markdown/pull/987/files). We all agreed to cancel it, since usually the strings is was attempting to migrate wouldn't render before anyway, ie `<wtf>`

Reply:
I'm also using AI to escape these, so we should be okay for the migration?